routes.go

```diff
@@ -311,6 +315,26 @@ func routeWithFlags(c *cli.Context, rt *fnmodels.Route) {
 			rt.Config = extractEnvConfig(c.StringSlice("config"))
 		}
 	}
+	if len(rt.Annotations) == 0 {
```

Comment:
at first look, it seems like a code duplication, no?

Reply:
Between this and the equivalent in apps.go?

Reply:
Yes.

Reply:
I agree. What suitable shared abstraction and package would you suggest? It seems odd to favour one or the other, and reducing locality by putting into the common package seemed unnecessary.

Reply:
i guess new function can find its home here: https://github.com/fnproject/cli/blob/master/common.go

Reply:
Great, thanks for the suggestion. I've pushed a change with the extracted function.